git_utils.py

```diff
@@ -164,26 +164,26 @@ def gitGetRemoteTags( url, debug = False, verbose = False ):
     tags = {}
     try:
         if verbose:
-            print "gitGetRemoteTags running: git ls-remote %s" % url
+            print("gitGetRemoteTags running: git ls-remote %s" % url)
         statusInfo = subprocess.check_output( [ 'git', 'ls-remote', url ], stderr=subprocess.STDOUT )
         for line in statusInfo.splitlines():
             if line is None:
                 break
-            tagSpecMatch = tagSpecRegExp.search( line )
+            tagSpecMatch = tagSpecRegExp.search( line.decode('utf-8') )
```

Comment:
Another way to accomplish the same is to change the subprocess call to specify `universal_newlines=True`, and then you get a `str` object out instead of a `bytes` object, I suspect this is harder for 2to3 to implement